The job **`x86_64-gnu-distcheck`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/14685161758/job/41212844294) [(plain)](https://github.com/rust-lang-ci/rust/commit/1c29b938bf040094c6a8f4e14e5c0980bea69d8c/checks/41212844294/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
file:.git/config remote.origin.url=https://github.com/rust-lang-ci/rust
file:.git/config remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
file:.git/config gc.auto=0
file:.git/config http.https://github.com/.extraheader=AUTHORIZATION: basic ***
file:.git/config branch.try.remote=origin
file:.git/config branch.try.merge=refs/heads/try
file:.git/config submodule.library/backtrace.active=true
file:.git/config submodule.library/backtrace.url=https://github.com/rust-lang/backtrace-rs.git
file:.git/config submodule.library/stdarch.active=true
file:.git/config submodule.library/stdarch.url=https://github.com/rust-lang/stdarch.git
file:.git/config submodule.src/doc/book.active=true
---
---- $DIR/issue-98690.rs - (line 7) stdout ----
error[E0463]: can't find crate for `foo`
##[error]  --> $DIR/issue-98690.rs:7:1
   |
LL | extern crate r#foo;
   | ^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0463`.
---
To only update this specific test, also pass `--test-args issues/issue-98690.rs`

error: 2 errors occurred comparing output.
status: exit status: 101
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/checkout/obj/build/tmp/distcheck/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" "/checkout/tests/rustdoc-ui/issues/issue-98690.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/checkout/vendor" "--sysroot" "/checkout/obj/build/tmp/distcheck/build/x86_64-unknown-linux-gnu/stage2" "--target=x86_64-unknown-linux-gnu" "--check-cfg" "cfg(test,FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "-o" "/checkout/obj/build/tmp/distcheck/build/x86_64-unknown-linux-gnu/test/rustdoc-ui/issues/issue-98690" "-A" "internal_features" "-Cdebuginfo=0" "--test" "--persist-doctests" "/../../" "-Z" "unstable-options"
--- stdout -------------------------------

running 1 test
test /checkout/tests/rustdoc-ui/issues/issue-98690.rs - (line 7) ... FAILED

failures:

---- /checkout/tests/rustdoc-ui/issues/issue-98690.rs - (line 7) stdout ----
error[E0463]: can't find crate for `foo`
##[error]  --> /checkout/tests/rustdoc-ui/issues/issue-98690.rs:7:1
   |
LL | extern crate r#foo;
   | ^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0463`.
---
test result: FAILED. 317 passed; 1 failed; 5 ignored; 0 measured; 0 filtered out; finished in 29.48s

Some tests failed in compiletest suite=rustdoc-ui mode=ui host=x86_64-unknown-linux-gnu target=x86_64-unknown-linux-gnu
Build completed unsuccessfully in 1:10:27
make: *** [Makefile:49: check] Error 1
Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 1:17:38
  local time: Sat Apr 26 22:41:15 UTC 2025
  network time: Sat, 26 Apr 2025 22:41:15 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile
+++ b/src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile
@@ -2,7 +2,7 @@
 # CentOS 7 has headers for kernel 3.10, but that's fine as long as we don't
 # actually use newer APIs in rustc or std without a fallback. It's more
 # important that we match glibc for ELF symbol versioning.
-FROM centos:7
+FROM ghcr.io/rust-lang/centos:7
 
 WORKDIR /build
 

--- a/src/ci/github-actions/jobs.yml
+++ b/src/ci/github-actions/jobs.yml
@@ -213,7 +213,7 @@ auto:
   - name: dist-x86_64-linux
     env:
       CODEGEN_BACKENDS: llvm,cranelift
-    <<: *job-linux-16c
+    <<: *job-linux-36c-codebuild
 
   - name: dist-x86_64-linux-alt
     env:

